source/keyboard-shortcuts.txt

```diff
@@ -194,35 +194,35 @@ Keyboard shortcuts enable you to easily navigate |compass-short|.
    * - Web Shell
      - Deletes the next character
      - ``Ctrl`` + ``D``
-     - ``Cmd`` + ``D``
+     - ``Ctrl`` + ``D``
 
    * - Web Shell
      - Moves the cursor to the end of the line
      - ``Ctrl`` + ``E``
-     - ``Cmd`` + ``E``
+     - ``Ctrl`` + ``E``
 
    * - Web Shell
      - Moves the cursor forward one character
      - ``Ctrl`` + ``F``
-     - ``Cmd`` + ``F``
+     - ``Ctrl`` + ``F``
 
    * - Web Shell
      - Erases one character, similar to hitting backspace
      - ``Ctrl`` + ``H``
      - ``Cmd`` + ``H``
 
    * - Web Shell
-     - Clears the screen, similar to the clear command.
+     - Clears the screen, similar to the clear command
      - ``Ctrl`` + ``L``
      - ``Cmd`` + ``L``
 
    * - Web Shell
-     - Swap the last two characters before the cursor.
+     - Swap the last two characters before the cursor
+     - ``Ctrl`` + ``T``
      - ``Ctrl`` + ``T``
-     - ``Cmd`` + ``T``
 
    * - Web Shell
-     - Changes the line to uppercase.
+     - Changes the line to uppercase
      - ``Ctrl`` + ``U``
      - ``Cmd`` + ``U``
```

Comment:
This actually does not seem to work. Maybe let's drop it for now.

Reply:
Removed.